source/frontend/src/hooks/pims-api/useApiContacts.ts

```diff
@@ -21,28 +21,29 @@ export const useApiContacts = () => {
   return React.useMemo(
     () => ({
       getContacts: (params: IPaginateContacts | null) =>
-        api.get<IPagedItems<IContactSearchResult>>(
+        api.get<ApiGen_Base_Page<IContactSearchResult>>(
           `/contacts/search?${params ? queryString.stringify(params) : ''}`,
         ),
       // This endpoint returns contact data in read-only form, including formatting some fields; e.g. full name = first + middle + last
       getContact: (id: string) => api.get<IContact>(`/contacts/${id}`),
       // This is different than getContact above. This endpoints returns person data that can be edited in a form
-      getPerson: (id: number) => api.get<IEditablePerson>(`/persons/${id}`),
+      // getPerson: (id: number) => api.get<IEditablePerson>(`/persons/${id}`),
```

Comment:
hmm? if we don't need this it should be removed not commented out.

Reply:
Yep, should be removed now that the person retrieval is standard